blackboxtest/nightwatch-tests/ssr.js

```diff
@@ -0,0 +1,44 @@
+const http = require('http');
+
+module.exports = {
+    'Ensure all of the standard routes return HTTP 200': (browser) => {
+        browser.end(); // Not needed.
+        [
+            '/',
+            '/trending',
+            '/created',
+            '/hot',
+            '/promoted',
+						'/about.html',
+						'/faq.html',
+						'/login.html',
+						'/privacy.html',
+						'/support.html',
+						'/tos.html',
+						'/change_password',
+						'/create_account',
+						'/approval',
+						'/recover_account_step_1',
+						'/recover_account_step_2',
+						'/waiting_list.html',
+						'/market',
+						'/~witnesses',
+						'/submit.html',
+        ].forEach(path => testHttpGet('localhost', 8080, path, browser));
```

Comment:
Does this not need some callback? Or some eqvl of `test.plan(numUrls)`

(ps indents are wonky)

Reply:
nope, this works! fixing indents now.

Reply:
What happens if the get request to one of the urls never returns?

Reply:
our test suite would hang!

we could do a timeout handler for `http` -- actually that is a good idea; i'll set it up to a short-ish limit since a slow rendering page is a bad sign anyways

Reply:
done; applied a timeout of 5000ms